src/swiftTextStyles.js

```diff
@@ -7,8 +7,12 @@ var swiftColors = require('./swiftColors.js')
 
 function getTextStylesSwiftFileContent(context, textStyles) {
   var code = getSwiftFontImportCode(context) + "\n\n";
+  code += "//swiftlint:disable file_length\n";
+  code += "//swiftlint:disable identifier_name\n\n";
   code += getTextStylesSwiftSnippet(context,textStyles, true) + "\n\n";
   code += getSwiftConvenienceStringExt();
+  code += "\n\n//swiftlint:enable identifier_name\n";
+  code += "//swiftlint:enable file_length\n";
```

Comment:
I would rather not include swiftlint comments by default as Swiftlint is not used by everyone and its usage greatly varies. Nonetheless, it could potentially be done in another PR where we would add a new option in the extension's settings.